API.md

```diff
@@ -58,9 +76,30 @@ Example JSON Response
   "surveyId": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87"
 }]
 ```
+
+### Get collection instrument by search string with limit
+
+* `GET /collection-instrument-api/1.0.2/collectioninstrument?searchString={"RU_REF":%20"test_ru_ref"}&limit=1`
```

Comment:
can we change searchString to classifiers for consistency with the post https://github.com/ONSdigital/ras-collection-instrument/pull/70/files#diff-88333ba11114e014e87198d04680144dR25